src/test/java/ua/utility/kfsdbupgrade/AppTestBase.java

```diff
@@ -0,0 +1,36 @@
+package ua.utility.kfsdbupgrade;
+
+import java.io.File;
+
+/**
+ * Created by S.G. Skinner on 2017-05-03
+ */
+public abstract class AppTestBase {
+    protected static final String DEFAULT_PROP_FILE =
```

Comment:
We can use the DEFAULT_PROPERTIES_FILE From the App.java, any reason to hard code the same path here?